ios/Sources/AboutFeature/AboutScreen.swift

```diff
@@ -1,16 +1,129 @@
+import ComposableArchitecture
 import SwiftUI
 import Styleguide
 
 public struct AboutScreen: View {
-    public init() {}
+
+    private let store: Store<AboutState, AboutAction>
+    
+    public init(store: Store<AboutState, AboutAction>) {
+        self.store = store
+        let fontDescriptor = UIFont.preferredFont(forTextStyle: .subheadline).fontDescriptor.withSymbolicTraits(.traitBold)!
+        let font = UIFont(descriptor: fontDescriptor, size: 0)
+        UISegmentedControl.appearance().selectedSegmentTintColor = UIColor(AssetColor.secondary.color)
+        UISegmentedControl.appearance().backgroundColor = UIColor(AssetColor.Background.secondary.color)
+
+        UISegmentedControl.appearance().setTitleTextAttributes([
+            .font: font,
+            .foregroundColor: UIColor(AssetColor.Base.primary.color)
+        ], for: .normal)
+
+        UISegmentedControl.appearance().setTitleTextAttributes([
+            .font: font,
+            .foregroundColor: UIColor(AssetColor.Base.white.color),
+        ], for: .selected)
+    }
 
     public var body: some View {
-        Text(L10n.AboutScreen.title)
+        NavigationView {
+            VStack {
+                WithViewStore(store) { viewStore in
+                    Picker(
+                        "",
+                        selection:
+                            viewStore.binding(
+                                get: { $0.selectedType },
+                                send: { .selectedPicker($0) }
+                            )
+                    ) {
+                        ForEach(SelectedType.allCases, id: \.self) { (type) in
+                            Text(type.title).tag(type)
+                        }
+                    }
+                    .pickerStyle(SegmentedPickerStyle())
+                    .padding(.top, 20)
+                    .padding(.horizontal)
+                    .padding(.bottom, 10)
+                    Divider()
+                        .foregroundColor(AssetColor.Separate.contents.color)
+                    switch viewStore.selectedType {
+                    case .staff:
+                        ScrollView(.vertical) {
+                            LazyVStack(alignment: .leading, spacing: 40) {
```

Comment:
I think this spacing should be `24`.
```suggestion
                            LazyVStack(alignment: .leading, spacing: 24) {
```

<img width="279" alt="スクリーンショット 2021-06-06 20 13 08" src="https://user-images.githubusercontent.com/30540303/120922372-b4f95300-c703-11eb-8cc1-01a251c78eb0.png">